gateway/platforms/servlet/src/main/java/io/apiman/gateway/platforms/servlet/connectors/HttpConnectorFactory.java

```diff
@@ -30,18 +35,27 @@
  * @author eric.wittmann@redhat.com
  */
 public class HttpConnectorFactory implements IConnectorFactory {
-    
+
+    // Standard auth
+    private SSLSessionStrategy standardSslStrategy;
+    // 2WAY auth (i.e. mutual auth)
+    private SSLSessionStrategy mutualAuthSslStrategy;
+    private Map<String, String> config;
+
     /**
      * Constructor.
+     * @param config map of configuration options
      */
-    public HttpConnectorFactory() {
+    public HttpConnectorFactory(Map<String, String> config) {
```

Comment:
Given that the config will be on IConnectorFactory (e.g. `apiman-gateway.components.IConnectorFactory.allowSelfSigned=true`) perhaps I should also namespace the keys to be something like apiman-gateway.components.IConnectorFactory.tls.allowSelfSigned=true